### ISSUE TITLE
[Kommentierung 1.5.3] Hinweistext unter der Anforderung KOM-LE-A_2048-01

### ISSUE THREAD

**Issue by CLAM01** [gematik/api-kim#47]
![image](https://github.com/gematik/api-kim/assets/115494634/eb43433f-2417-44d0-b73e-f3093c78ced1)

Verstehe ich es richtig, dass bei der Prüfung der Header-Elemente (inhalt nicht kennzeichner) from, sender, reply-to, to und cc alle Elemente geprüft werden sollen?
Also DisplayName und spec-addr? Oder soll nur die spec-addr gemäß RfC5322 geprüft / verglichen werden?

**Comment by gem-cp** [comment]
Die Anforderung besagt, dass die Werte der Header-Elemente in der Integritätsprüfung enthalten sind. Dazu gehört auch der display-name, wie in der der Address Specification (https://www.rfc-editor.org/rfc/rfc5322.html#section-3.4) beschrieben.
Aus meiner Sicht: Wenn der display-name nicht in der Integritätsprüfung berücksichtigt wird, könnte man dem Empfänger einen falschen Absender vortäuschen.

Leider steht im Hinweistext eine Reference auf Kapitel 3.4 des RFC mit der Überschrift "Address Specification". In Kapitel 3.4.1 heißt die Überschrift "Addr-Spec Specification" so wie im Hinweistext; ist also ungenau formuliert. Das können wir gern korrigieren.

Gibt es Konsens, dass display-name bei der Integrationsprüfung zu berücksichtigen ist?

**Comment by CLAM01** [comment]
Ja, das wäre wünschenswert. Genau die existierende Überschrift schafft die Verwirrung.
